src/Common/tests/System/Net/PlatformDetection.Networking.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static partial class PlatformDetection
```

Comment:
Is there any reason for not using the existing Capabilities.cs instead of PlatformDetection?